conduct/code-of-conduct.md

```diff
@@ -60,7 +53,7 @@ While this code of conduct applies to spaces associated with `<EVENT>`, if you a
 
 In order to protect volunteers from abuse and burnout, we reserve the right to reject any report we believe to have been made in bad faith. Reports intended to silence legitimate criticism may be deleted without response.
 
-We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them if we believe that doing so will increase the safety of `<EVENT>` participants or the general public. We will not name harassment victims without their affirmative consent.
```

Comment:
Can we retain this:

> privately warn third parties about them if we believe that doing so will increase the safety of `<EVENT>` participants or the general public

I'm having trouble figuring out a defensible argument for removing this. I can see an argument for not naming publicly, but sharing privately to protect others seems warranted. Maybe it doesn't matter though b/c a black list will emerge either way